src/main/java/core/basesyntax/Plane.java

```diff
@@ -8,4 +8,55 @@
 
 public class Plane {
 
+    private String name;
+    private int maxSpeed;
+    private int maxHeight;
+    private int numberOfPassengers;
+    private String manufacturer;
+
+    public Plane(PlaneBuilder planeBuilder) {
```

Comment:
we should be able to call this constructor from inside this class only
```suggestion
    private Plane(PlaneBuilder planeBuilder) {
```